### PR TITLE
Map

### PR DESCRIPTION
**Pull request by foxwellm**
Since you have an apple phone vs my android, try to test that it's working for me please

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
...
     this.state = {
-      currentPage: 'Login'
+      currentPage: 'Home',
+      currentLatitude: null,
+      currentLongitude: null
     }
   }
 
+  componentDidMount() {
+    this.watchID = navigator.geolocation.watchPosition(
```
Comment (foxwellm) [inline App.js:19]: watchID is a global variable that the watchPosition method will keep updated one app starts to keep track of users' location

File: App.js [hunk 2]
```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
...
+        });
+      },
+      error => console.log(error),
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 0, distanceFilter: 1 }
+    );
+  };
+
+  componentWillUnmount() {
+    navigator.geolocation.clearWatch(this.watchId);
+  };
```
Comment (foxwellm) [inline App.js:34]: End tracking once app is closed

File: App.js [hunk 3]
```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
...
   constructor() {
-    super()
+    super();
     this.state = {
-      currentPage: 'Login'
+      currentPage: 'Home',
+      currentLatitude: null,
+      currentLongitude: null
     }
   }
```
Comment (hlhartley) [inline App.js:17]: Can you please explain this in person when you have time?

File: App.js [hunk 4]
```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
...
+    this.watchID = navigator.geolocation.watchPosition(
+      position => {
+        const { latitude: currentLatitude, longitude: currentLongitude } = position.coords;
+        this.setState({
+          currentLatitude,
+          currentLongitude
+        });
+      },
+      error => console.log(error),
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 0, distanceFilter: 1 }
```
Comment (foxwellm) [inline App.js:28]: These arguments are for watchPosition,
enableHighAccuracy is for GPS tracking vs wifi
timeout after 20s
maximumAge:0 is so it updates whenever possible without using cached value
distanceFilter:1 is tracking any change greater than 1m.

Comment (hlhartley) [reply App.js:28]: Ok thanks

**Review by hlhartley — APPROVED** [review]

**Pull request merged**